index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Agasha Bornittah Portifolio</title>
+    <link rel="stylesheet" href="assets/fonts/css/all.min.css">
+    <link rel="stylesheet" href="assets/fonts/css/brands.min.css">
+    <link rel="stylesheet" href="assets/fonts/css/fontawesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+   <div class="container">
+       <div class="nav">
+        <h1>Welcome</h1>
+        <div class="nav-menu" >
+            <ul>
+                <li>
+                    <a href="">Portifolio</a>
+                </li>
+                <li>
+                    <a href="">About</a>
+                </li>
+                <li>
+                    <a href="">Contact</a>
+                </li>
+            </ul>
+        </div>
+        <a href="" class="menu-bars">
+            <i class="fas fa-bars"></i>
+        </a>
+       </div>
+        <div class="section">
```

Comment:
- [ ] Kindly change this `div` tag to a section tag, kindly align all content to look like the Figma design, You can use margin and padding, check the screenshot below 👍🏿 
 
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![aaaa](https://user-images.githubusercontent.com/69638013/164414497-57b4b88e-f828-4b20-8b64-32043aca165f.png)  |  ![bbb](https://user-images.githubusercontent.com/69638013/164414808-f69a8963-9b79-4187-b2be-18f78b272208.png)